packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -67,7 +68,7 @@ abstract class CameraPlatform extends PlatformInterface {
   }
 
   /// Captures an image and saves it to [path].
```

Comment:
Would be good to update the documentation. Maybe something like:
```suggestion
  /// Captures an image and returns it as a `XFile`.
```
```dart